docs/src/constrained/irregular_surrogates.md

```diff
@@ -0,0 +1,25 @@
+# Irregular Timeseries Surrogates
+
+
+To derive a surrogate for irregular time series we can use surrogate methods which are irrespective of the time axis like [`RandomShuffle`](@ref) or [`BlockShuffle`](@ref)
```

Comment:
The statement _"... surrogate methods which are irrespective of the time axis"_ doesn't quite make sense. Maybe it would be better to say something like _"To derivate a surrogate for time series which are irregularly sampled, we can use surrogate methods that does not explicitly require regularly sampled time series, for example [`RandomShuffle`](@ref) or [`BlockShuffle`](@ref). Alternatively, we need to use algorithms that take the irregularity of the time axis into account._